### ISSUE TITLE
New release to bower eliminates previous versions

### ISSUE THREAD

**Issue by PjEdwards** [carrot/share-button#252]
Any project depending on a version other than 1.0.3 will experience `bower install` failure.

**Comment by hhsnopek** [comment]
I can get around to this tonight, I'll get them all up for yah :+1:

**Comment by PjEdwards** [comment]
@hhsnopek Fantastic news! Thanks!

**Comment by mmautner** [comment]
:+1: still not there :'(

**Comment by hhsnopek** [comment]
totally forgot!! I'll try my best to get this done later today or tomorrow

**Comment by Pirozek** [comment]
Please can you return the old repo back? I love your share button, but now I can't use it in my projects (i don't use 1.0.3 version)

**Comment by hhsnopek** [comment]
Which version are you looking to pull down my plan is to just gave 1.0.x up therr



--
Henry Snopek

**Comment by Pirozek** [comment]
Well we are using 0.5 :) Updating to newer version is in TODO list :-D

**Comment by PjEdwards** [comment]
Yeah..... :) we are on 0.4.3. Updating to newer version is on the TODO list

**Comment by hhsnopek** [comment]
haha alright - I'll just get them all up

**Comment by Pirozek** [comment]
Version 1+ is better, but for my legacy projects its like "Its working, dont touch it!", so I don't want to change it to newer button (self-preservation purposes :-D)

**Comment by rohanpatel2602** [comment]
@hhsnopek Hey Henry, any update on this? I require 0.5.0 myself

**Comment by Pirozek** [comment]
+1
